runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -86,6 +87,8 @@ public class FishingPlugin extends Plugin
 
 	private Instant trawlerStartTime;
 
+	private static final Pattern FISHING_CATCH_REGEX = Pattern.compile("(You catch (a|an) )|(You catch some)|(Your cormorant returns with its catch.)|(You catch .* Karambwanji)");
```

Comment:
this is just `You catch (a|an|some) |Your cormorant returns with its catch.|You catch .* Karambwanji`. You don't need to add unnecessary capture groups for everything.